indico/modules/rb_new/client/js/modules/calendar/Calendar.jsx

```diff
@@ -167,12 +179,24 @@ class Calendar extends React.Component {
                         </div>
                     </Container>
                 </Grid.Row>
-                <Route exact path="/calendar/:roomId/details" render={roomPreloader((roomId) => (
+                <Route exact path="/calendar/rooms/:roomId/details" render={roomPreloader((roomId) => (
                     <RoomDetailsModal roomId={roomId} onClose={this.closeModal} />
                 ), fetchRoomDetails)} />
                 <Route exact path="/calendar/:roomId/book" render={roomPreloader((roomId) => (
                     <BookFromListModal roomId={roomId} onClose={this.closeModal} />
                 ), fetchRoomDetails)} />
+                <Route exact path="/calendar/bookings/:bookingId/details" render={({match: {params: {bookingId}}}) => {
+                    return (
```

Comment:
you don't need an explicit `return` here. You can replace enclosing curly braces with brackets and it will return the component automatically